pkg/alerts/alertsHandler/cronJobHandler.go

```diff
@@ -242,7 +273,9 @@ func evaluateLogAlert(alertToEvaluate *alertutils.AlertDetails, job gocron.Job)
 
 	isAlertConditionMatched := evaluateConditions(serResVal, &alertToEvaluate.Condition, alertToEvaluate.Value)
 
-	err = handleAlertCondition(alertToEvaluate, isAlertConditionMatched, "")
+	alertDataMessage := fmt.Sprintf("Check the logs for the Alert Query: %v", getLogsQueryLinkForTheAlert(alertToEvaluate, timeRange))
```

Comment:
nit: `Check the logs for the Alert Query:` is a little unclear to me, but you can keep it if you want. I think it's because to me `Check the logs` sounds like you're telling me to check the logs file rather than to run the query on the UI. Maybe something like `Run the Alert Query` or `View the query results` would be better

Reply:
I think I will change it to: `View the Query Results`.